src/ShallowWrapper.js

```diff
@@ -206,6 +206,22 @@ export default class ShallowWrapper {
   }
 
   /**
+   * Whether or not a given react element exists in the shallow render tree.
+   *
+   * Example:
+   * ```
+   * const wrapper = shallow(<MyComponent />);
+   * expect(wrapper.contains(<div className="foo bar" />)).to.equal(true);
+   * ```
+   *
+   * @param {ReactElement} node
+   * @returns {Boolean}
+   */
+  equals(node) {
+    return nodeEqual(this.node, node);
```

Comment:
can you wrap this inside a `return this.single(() => nodeEqual(this.node, node));`